examples/VASP/water-1-npt-md-vasp/ipi-vasp.xml

```diff
@@ -0,0 +1,54 @@
+<simulation verbosity="high">
+  <output prefix="simulation">
+      <properties stride='1' filename='out'>
+        [ step, time{picosecond}, conserved, temperature{kelvin}, potential{electronvolt}, pressure_md{gigapascal} ]
+      </properties>
+      <trajectory filename='fc' stride='1'>f_centroid</trajectory>
+      <trajectory filename='xc' stride='1'>x_centroid{angstrom}</trajectory>
+      <properties filename='cell' stride='1'>[cell_h{angstrom}]</properties>
+      <properties filename='stress' stride='1'>[virial_md{bar}]</properties>
+<checkpoint filename="chk" stride="2" overwrite="true"/>
+  </output>
+  <prng><seed> 3924</seed></prng>
+  <total_steps>100</total_steps>
+    <ffsocket mode='inet' name='aaa'>
+         <port> 23333 </port>
+         <address> localhost </address>
+   </ffsocket>  
+<system>
+    <initialize nbeads="1">
+      <file mode="xyz" units="angstrom"> h2o.xyz </file>
+      <cell mode="manual" units="angstrom"> [ 8,0.5,0.2, 0.0,9,0.6, 0.0,0.0,7 ] </cell>
```

Comment:
could you modify your inputs to include the units in the files instead of the xml inputs? it is what we recommend currently to avoid double-conversions or wrong-conversions.

Reply:
Dear Mariana, 

OK, I didn't know there's a new feature that units can be specified in the xyz file in i-pi. I can add the units to the xyz files.

Wei